### PR TITLE
Release for mixed session

### PR DESCRIPTION
**Pull request by tungmhoang**
### Summary

- Update content of mixed session guide. Preview link: http://10.2.7.208/manual-testing/start-a-mixed-session

### Related PRs, issues, or features (optional)

- https://kobiton.atlassian.net/browse/KOB-39139

### Metadata

- [x] Adds new file(s)
- [x] Edits existing file(s)
- [x] Removes file(s)

### PR contributor checklist

- [x] My PR follows the [Kobiton Docs contributor guidelines](https://github.com/kobiton/docs/blob/main/CONTRIBUTE.adoc), meaning:

    - My content contains correct spelling and grammar.
    - My directories and files are [named](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-and-file-names) and [structured](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-structure) correctly.
    - My style and voice follows the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/brand-voice-above-all-simple-human).
    - I added all new pages to their section's [`nav.adoc` file](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#configure-navigation-in-navadoc).
    - I removed all localizations (like `en-us`) from my URLs.

### CODE REVIEW

File: docs/modules/automation-testing/pages/start-a-mixed-session.adoc [hunk 1]
```diff
@@ -1,7 +1,13 @@
 = Start a mixed session
 :navtitle: Start a mixed session
 
-Learn how to start a mixed session so you can view your Appium script running on your device and fix issues in realtime.
+Learn how to start a mixed session, so you can view your Appium script running on your device and fix issues in realtime.
+
+== What is a mixed session?
+
+A mixed session is where you can use the feature of a manual session to observe what is happening when running an Appium automation script.
```
Comment (erincbailey) [inline docs/modules/automation-testing/pages/start-a-mixed-session.adoc:8]: A mixed session allows you to observe what occurs during the execution of an Appium automation script through a manual session.

File: docs/modules/automation-testing/pages/start-a-mixed-session.adoc [hunk 2]
```diff
@@ -13,20 +19,58 @@ image:automation-testing:launch-device-context.png[width=800,alt="A context of l
...
 
-* In a Mixed session, both the Manual and Automation session must be initiated by the same user.
+* In a mixed session, both the Manual and Automation session must be initiated by the same user.
 
 * Mixed session are only available for Kobiton 4+ devices.
 
-* Mixed sessions can only be initiated with Appium automation tests; other automation frameworks, e.g. Espresso, XCUITest, are not currently supported.
+* Mixed sessions can only be initiated with Appium or Basic Appium 2 automation tests; other automation frameworks, e.g. Espresso, XCUITest, are not currently supported.
+
+* Manual actions in a mixed session is not yet available when a Basic Appium 2 script is running.
```
Comment (erincbailey) [inline docs/modules/automation-testing/pages/start-a-mixed-session.adoc:68]: Performing manual actions in a mixed session is not supported for Basic Appium 2 script execution.

File: docs/modules/automation-testing/pages/start-a-mixed-session.adoc [hunk 3]
```diff
@@ -13,20 +19,58 @@ image:automation-testing:launch-device-context.png[width=800,alt="A context of l
...
 
 * Session Explorer is not available for Mixed Sessions; i.e. it's not possible to view detailed steps and screenshots for Mixed Sessions.
 
 * Mixed sessions cannot be turned into test cases. They cannot be used for our AI functionality: Scriptless, generating Appium Scripts, self-healing.
 
-* When you have a Mixed Session on a Public Device, you will be charged minutes for the total time usage on the device; you will not be double-charged for an automation and a manual session.
+* When you have a mixed Session on a Public Device, you will be charged minutes for the total time usage on the device; you will not be double-charged for an automation and a manual session.
+
+* A mixed session counts as 1 parallel test session, although it includes a manual and an automation session.
```
Comment (erincbailey) [inline docs/modules/automation-testing/pages/start-a-mixed-session.adoc:76]: A mixed session counts as 1 parallel test session, even though it includes both a manual and an automation session.

**Review by erincbailey — CHANGES_REQUESTED** [review]
I was not able to review this in the "preview" site - but I looked through the changes. I made a couple of wording suggestions. Then you can publish. Thank you!

**Comment by tungmhoang** [comment]
@erincbailey please approve the PR so I can merge

**Review by erincbailey — APPROVED** [review]
approved

**Pull request merged**